lib/lograge.rb

```diff
@@ -116,7 +116,8 @@ def self.unsubscribe(component, subscriber)
 
   def self.setup(app)
     self.application = app
-    app.config.action_dispatch.rack_cache[:verbose] = false if app.config.action_dispatch.rack_cache
+    app.config.action_dispatch.rack_cache[:verbose] = false if app.config.action_dispatch.rack_cache &&
```

Comment:
I suggest moving the boolean check into a method to make the code a bit cleaner.